packages/beacon-node/src/util/address.ts

```diff
@@ -1,3 +1,3 @@
 export function isValidAddress(address: string): boolean {
-  return !!address && address.startsWith("0x") && address.length === 42;
+  return /^0x[0-9a-fA-F]{40}$/.test(address);
```

Comment:
not sure how much this is used, I think it may be faster to create a singleton regex object, so it can be reused each time isValidAddress is called.

Reply:
NodeJS should cache those when interpreting the code (in theory)